wordpress-mobile/WordPress-iOS
# Added analytics support to content extension

## Description
Fixes #9841, which calls for analytics events to track rich notifications engagement. To start, we'll track the launch of the content extension, which is triggered by a user force-touching on the _Short Look_ to evince the _Long Look_ for comment notifications.

## To test

- Verify that the branch builds & existing tests pass.
- While debugging the content extension, manually push a payload with a valid `note_id` value. Instructions for this can be found here: #9924 
- In the _Tracks Live View_, validate that events of type `wpios_notification_content_extension_launched` can be found.
